.gitignore

```diff
@@ -14,9 +14,5 @@ website/i18n/*
 website/static/aragon-ui
 website/*.lock
 
-# ignore files generated from repos
```

Comment:
I believe is better to allow edits on one place

Reply:
> I believe is better to allow edits on one place

Definitely, nice catch with the `custom_edit_url` option!